.vscode/launch.json

```
@@ -0,0 +1,23 @@
+{
+    "version": "0.2.0",
+    "configurations": [
+        {
+            "name": "Debug Marge",
+            "type": "python",
+            "request": "launch",
+            "cwd": "${workspaceFolder}",
+            "module": "marge",
+            "args": [
+                "--config-file=development-config.yaml"
+            ]
+        },
+        {
+            "name": "Debug tests",
```

Minor, but given that VS Code already displays "RUN AND DEBUG" next to this, perhaps it's fine to leave out "Debug" from the name here? Something like this:

```suggestion
            "name": "pytest",
```